UIKitDZ/UIKitDZ/SingInViewController.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  SingInViewController.swift
+//  UIKitDZ
+//
+//  Created by Maxim on 04.10.2022.
+//
+
+import UIKit
+/// Контроллер входа в приложение
+final class SingInViewController: UIViewController {
+    
+    // MARK: - IBOutlet
+    @IBOutlet weak var emailTextField: UITextField!
+    @IBOutlet weak var passwordTextField: UITextField!
+    
+    // MARK: - Private Properties
+    private var key = ""
+    
+    // MARK: - Life Cycle
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        transitionTextField()
+    }
+    
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        key = UserDefaults.standard.string(forKey: "emailTextField") ?? ""
+    }
+
+    // MARK: - IBAction
+    @IBAction func singInButton(_ sender: Any) {
```

Comment:
```suggestion
    @IBAction func singInAction(_ sender: Any) {
```
Если IBAction значит в конце метода пишем Action